ApplicationPattern.yaml

```diff
@@ -2631,8 +2631,389 @@ paths:
                     ipv-4-address: '1.1.3.13'
                 topology-application-port: 3013
       responses:
-        '204':
-          description: 'Client side for sending information about topology changes has been updated'
+        '200':
```

Comment:
changes are as expected. response code is now "200" and the responseBody schema is having the control-construct included.